libretroshare/src/services/p3discovery2.h

```diff
@@ -131,6 +133,7 @@ virtual void setGPGOperation(AuthGPGOperation *operation);
 	p3LinkMgr *mLinkMgr;
 	p3NetMgr  *mNetMgr;
 	p3ServiceControl *mServiceCtrl;
+    RsGixs *mGixs ;
```

Comment:
Better to take a reference (const?) RsGixs& gixs instead of a pointer in this case as it must not be null